gulp/styles.js

```diff
@@ -12,6 +14,19 @@ var wiredep = require('wiredep').stream;
 var _ = require('lodash');
 var rename = require("gulp-rename");
 
+
+// generate sprites
+gulp.task('sprites', function () {
+  return sprity.src({
+    src: './src/images/sprites/*/*.{png,jpg}',
```

Comment:
No because I do not want to take into account the images in the "middle" folder. mnoe-icon.png will be stored there and I would get a sprite of sprite #inception